OracleEDQ/12.2.1.3.0/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2014-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Enterprise Data Quality 12.2.1.3.0
+#
+# Base image of this dockerfile is Oracle Java 8 docker image.
+#
+# Pre-requisite to build oracle Oracle Enterprise Data Quality 12.2.1.3.0 image is to have
```

Comment:
Actually, the pre-requisite is to build the FMW Infrastructure image, not just the Java 8 image.

Reply:
Fixed, Changed to
# Base image of this dockerfile is Oracle FMW Infrastructure and Oracle Java 8 docker image.